app/DynamicsAdapter/Fams3Adapter.Dynamics/InsuranceClaim/SSG_SimplePhoneNumber.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Newtonsoft.Json;
+
+namespace Fams3Adapter.Dynamics.InsuranceClaim
+{
+    public class SSG_SimplePhoneNumber
+    {
+        [JsonProperty("ssg_phonenumber")]
+        public string PhoneNumber { get; set; }
+
+        [JsonProperty("ssg_phonenumbertype")]
+        public string Type { get; set; }
+
+        [JsonProperty("ssg_phoneextension")]
+        public string Extension { get; set; }
+
+        [JsonProperty("ssg_ICBCClaim")]
```

Comment:
We should check with dynamics people, i thought we are looking at linking this to any entity that may require it's own phone? It seems to be tied strictly to only icbc claim .

Reply:
currently, it is.